### PR TITLE
Homework10

### PR DESCRIPTION
**Pull request by YeaaaaH**

### CODE REVIEW

File: HW/src/main/java/homework10/Main.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    public static void main(String[] args) throws IOException {
+        getOutPut();
+    }
+    static void getOutPut() throws IOException {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+        String string = String.valueOf(bufferedReader.readLine());
+        bufferedReader.close();
+
+        String[] st = string.split("[^A-Za-z0-9]+");
+        List<String> list = Stream.of(st).collect(Collectors.toList());
```
Comment (Siril-O) [inline HW/src/main/java/homework10/Main.java]: Why you need list here, just create stream

File: HW/src/main/java/homework10/Main.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        Map<String, Long> map = new LinkedHashMap<>();
+        list.stream().
+                map(String::toLowerCase).
+                collect(Collectors.groupingBy(Function.identity(), Collectors.counting())).
+                entrySet().
+                stream().
+                sorted(Comparator.<Map.Entry<String, Long>, Long>
+                        comparing(Map.Entry::getValue).reversed()
+                        .thenComparing(Map.Entry::getKey)).limit(10).forEach(a -> System.out.println(a.getKey()));
```
Comment (Siril-O) [inline HW/src/main/java/homework10/Main.java]: good, format it so each operation is in new line

File: HW/src/main/java/homework10/Main.java (hunk 1 above)
Comment (YeaaaaH) [reply HW/src/main/java/homework10/Main.java]: done

File: HW/src/main/java/homework10/Main.java (hunk 2 above)
Comment (YeaaaaH) [reply HW/src/main/java/homework10/Main.java]: done

**Pull request merged**